src/Server/HTTPHandlerFactory.cpp

```diff
@@ -167,14 +179,17 @@ HTTPRequestHandlerFactoryPtr createHandlerFactory(IServer & server, Asynchronous
 HTTPRequestHandlerFactoryPtr createMetaStoreHandlerFactory(IServer & server, const std::string & name)
 {
     auto factory = std::make_shared<HTTPRequestHandlerFactoryMain>(name);
-    auto rest_handler = std::make_shared<HandlingRuleHTTPHandlerFactory<RestHTTPRequestHandler>>(server, "metastore");
-    rest_handler->attachNonStrictPath("/proton/metastore");
-    factory->addHandler(rest_handler);
+    for (const auto * prefix : {"timeplusd", "proton"})
```

Comment:
So both `timeplusd/metastore` and `proton/metastore` will be there for a while, right?

Reply:
Yes